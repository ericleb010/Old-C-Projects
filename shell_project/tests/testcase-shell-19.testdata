gabby gook=simple
love = bleh
echo ${ cake }
