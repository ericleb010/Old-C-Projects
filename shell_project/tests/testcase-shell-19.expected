ERROR: gabby gook is not a valid variable name. Must not contain characters other than alphanumerics and '_' characters.
ERROR: love  is not a valid variable name. Must not contain characters other than alphanumerics and '_' characters.
ERROR: expected a '}' within statement.

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 
